site/specifications/default.md

```diff
@@ -64,7 +64,7 @@ specifications for drivers, protocols, SCPI, and VXI*plug&play*.
 |IVI-4.4: IviDCPwr Class Specification|[Word](../downloads/Class%20Specifications/IVI-4.4_DCPwr_2015-09-24.doc)|[PDF](../downloads/Class%20Specifications/IVI-4.4_DCPwr_2015-09-24.pdf)|
 |IVI-4.5: IviACPwr Class Specification|[Word](../downloads/Class%20Specifications/IVI-4.5_ACPwr_2011-03-11.doc)|[PDF](../downloads/Class%20Specifications/IVI-4.5_ACPwr_2011-03-11.pdf)|
 |IVI-4.6: IviSwtch Class Specification|[Word](../downloads/Class%20Specifications/IVI-4.6_Swtch_2017-03-03.doc)|[PDF](../downloads/Class%20Specifications/IVI-4.6_Swtch_2017-03-03.pdf)|
-|IVI-4.7: IviPwrMeter Class Specification|[Word](../downloads/Class%20Specifications/IVI-4.7_PwrMeter_2016-10-14.doc)|[PDF](../downloads/Class%20Specifications/IVI-4.7_PwrMeter_2016-10-14%20\(2\).pdf)|
+|IVI-4.7: IviPwrMeter Class Specification|[Word](../downloads/Class%20Specifications/IVI-4.7_PwrMeter_2016-10-14.doc)|[PDF](../downloads/Class%20Specifications/IVI-4.7_PwrMeter_2016-10-14%20(2).pdf)|
```

Comment:
Good fix!

Reply:
Better fix would be to correct the name of the file.  I'm pretty sure the '2' is just because someone downloaded a second copy and browser added the number.